backend/local_graphql_server.py

```diff
@@ -72,7 +73,7 @@ def request_context(headers, mock=False):
 
     for group in groups:
         with engine.scoped_session() as session:
-            TenantPolicy.attach_group_tenant_policy(
```

Comment:
Instead of hardcoding `dataall` we could use `TENANT_NAME = 'dataall'` from `TenantPolicyService`  currently it is defined in repositories, but it might be better placed in service